test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,7 +9,15 @@
 import org.junit.Test;
 
 public class UtilsTest {
-
+    
+    @Test
+    public void isAnyNull() {
+        assertFalse(Utils.isAnyNull());
+        assertTrue(Utils.isAnyNull((Object) null));
```

Comment:
Perhaps you can use comments to separate different test cases for better readability? For example,
// Empty list
assertFalse(Utils.isAnyNull());

// Non-empty list with nulls
assertTrue(...);
...